inc/notificationtemplate.class.php

```diff
@@ -577,4 +577,33 @@ function cleanDBonPurge() {
       $queued->deleteByCriteria(['notificationtemplates_id' => $this->fields['id']]);
    }
 
+   function prepareInputForClone($input) {
+      parent::prepareInputForClone($input);
+      $input['name'] = $input['name'] . ' (clone)';
+      return $input;
+   }
+
+   function post_clone($source, $history) {
```

Comment:
I may be wrong, but I think it should be done by using the `Glpi\Features\Clonable` trait and adding `NotificationTemplateTranslation::class` in a `public function getCloneRelations()` method.